release/connect-streams.js

```diff
@@ -25456,7 +25489,7 @@
 
   connect.core = {};
   connect.core.initialized = false;
-  connect.version = "1.6.6";
+  connect.version = "1.6.7";
```

Comment:
can we bump this to match the version in package.json? I think I missed bumping this when I released the hotfix as 1.6.7.

Reply:
good catch, thanks